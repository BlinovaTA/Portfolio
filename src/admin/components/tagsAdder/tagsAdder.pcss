.tags {
  display: flex;
  flex-wrap: wrap;
  margin-top: 20px;
}

.tag {
  background: #F3F6F9;
  border-radius: 40px;
}

.tag-wrapper {
  margin-right: 10px;
  margin-bottom: 10px;
}