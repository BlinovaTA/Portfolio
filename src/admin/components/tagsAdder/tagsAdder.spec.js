import tagsAdder from "./tagsAdder.vue";
import { mount } from "@vue/test-utils";

let wrapper;

beforeEach(() => {
  wrapper = mount(tagsAdder);
});

afterEach(() => {
  wrapper.destroy();
});

describe("tags adder tests", () => {
  it("tags adder snapshot", () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it("tags string to array", async () => {
    await wrapper.setProps({
      tags: "tag1,tag2,tag3",
    });

    const tags = wrapper.findAll("li");

    expect(tags.length).toBe(3);
  });
});
