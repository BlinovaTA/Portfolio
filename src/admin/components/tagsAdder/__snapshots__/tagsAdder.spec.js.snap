// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tags adder tests tags adder snapshot 1`] = `
<div
  class="tags-adder-component"
>
  <label
    class="input input_labeled"
  >
    <div
      class="title"
    >
      Добавление тега
    </div>
     
    <input
      class="input__elem field__elem"
    />
     
    <div
      class="input__error-tooltip"
    >
      <div
        class="input__error-tooltip-container"
      >
        
      </div>
    </div>
  </label>
  <ul
    class="tags"
  >
    <!---->
  </ul>
</div>
`;
